### PR TITLE
WIP: Various Bug Fixes

### PR DESCRIPTION
**Pull request by timthirion**
- [x] #2—Default to quad view
- [x] #11—visualization module should be namespaced
- [ ] #10—Add cascading delete to the UI
- [ ] #7—Painting resets the 3D view

### CODE REVIEW

**Comment by timthirion** [comment]
@floryst I'm going to try to pick off some bugs from your list. We can merge whenever you feel like there's enough.

**Comment by floryst** [comment]
Sounds good to me! I'll do some bug hunting to see what else I can find.

**Comment by timthirion** [comment]
@floryst Can you spot check the last commit? I think I've got most of the namespacing done but would be good to know what I missed.

File: src/components/ModelBrowser.vue [hunk 1]
```diff
@@ -77,7 +77,7 @@ export default {
         colorBy: { array, location },
       });
     },
-    ...mapActions(['removeData', 'setModelColorBy']),
+    ...mapActions('visualization', ['removeData', 'setModelColorBy']),
```
Comment (floryst) [inline src/components/ModelBrowser.vue:80]: You can actually get rid of `removeData` for now. It turns out I don't have any code to delete models yet.

File: src/components/PatientBrowser.vue [hunk 2]
```diff
@@ -371,7 +371,7 @@ export default {
       return '';
     },
 
-    ...mapActions(['selectBaseImage', 'updateScene', 'removeData']),
+    ...mapActions('visualization', ['selectBaseImage', 'updateScene', 'removeData']),
```
Comment (floryst) [inline src/components/PatientBrowser.vue:374]: We want to use the top-level `removeData` action, and that should call into the visualization model's removeData, so we should use the non-namespaced version of that action.

File: src/components/VtkTwoView.vue [hunk 3]
```diff
@@ -70,13 +70,14 @@ export default {
     ...mapState({
       baseImage: 'selectedBaseImage',
       baseImageExists: (state) => state.selectedBaseImage !== NO_SELECTION,
-      worldOrientation: (state) => state.visualization.worldOrientation,
-      windowing: (state) => state.visualization.windowing,
-      resizeToFit: (state) => state.visualization.resizeToFit,
-      slices: (state) => state.visualization.slices,
-      dataIndex: (state) => state.data.index,
```
Comment (floryst) [inline src/components/VtkTwoView.vue:77]: Don't forget to re-add `dataIndex` to the top-level mapState.

**Review by floryst — CHANGES_REQUESTED** [review]
It's a good first pass! We should also edit the `removeData` in `datasets.js` to call `dispatch('visualization/removeData', dataID)` before we remove annotation data.

File: src/components/PatientBrowser.vue (hunk 2 above)
Comment (timthirion) [reply src/components/PatientBrowser.vue:374]: Further evidence to clarify `removeData`!

Comment (floryst) [reply src/components/PatientBrowser.vue]: Agreed! We should rename other uses of `removeData` to be something more specific to that module.

File: src/components/VolumeRendering.vue [hunk 4]
```diff
@@ -118,10 +118,8 @@ export default {
   },
 
   computed: {
-    ...mapState({
-      baseImage: 'selectedBaseImage',
-      colorBy: (state) => state.visualization.colorBy,
-    }),
+    ...mapState({ baseImage: 'selectedBaseImage' }),
+    ...mapState('visualization', ['colorBy']),
     ...mapGetters(['baseImagePipeline', 'baseImageColorPreset']),
```
Comment (floryst) [inline src/components/VolumeRendering.vue:123]: These are part of the visualization namespace (check out the getters in `src/store/visualization.js`).

**Review by floryst — CHANGES_REQUESTED** [review]
For renaming, we can rename the other instances of `removeData` in a separate PR.

**Comment by timthirion** [comment]
@floryst After all the fixups for namespacing the visualization module, most things generally appear to be functioning. Is there specific functionality I should test?

**Comment by floryst** [comment]
So long as things are working, I'm good to merge this. We can fix things up later.

**Review by floryst — APPROVED** [review]

**Pull request merged**